sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -716,4 +716,8 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("set / set -v") {
+    checkExistence(sql("set"), true, "env:", "system:")
```

Comment:
SPARK_TESTING is not in sys.env nor sys.props while running the test cases. Any suggestions to add the test cases for that?